packages/visx-demo/src/sandboxes/visx-delaunay-voronoi/Example.tsx

```diff
@@ -71,7 +73,7 @@ function Example({ width, height, margin = defaultMargin }: VoronoiProps) {
 
           const closest = voronoiDiagram.delaunay.find(point.x, point.y);
           // find neighboring polygons to hightlight
-          if (closest && data[closest].id !== hoveredId) {
+          if (data[closest].id !== hoveredId) {
```

Comment:
is `find` guaranteed to return something? if `closest` could be `undefined`, this would then throw an error when evaluating `data[closest].id`

Reply:
In the `voronoiDiagram.delaunay.find(point.x, point.y);` line, the `find` method of the Delaunay class from `d3-delaunay` is being used, which is defined as follows in `@types/d3-delaunay`, guaranteeing that a number type will always be returned according to the type definition:
```ts
/**
 * Returns the index of the input point that is closest to the specified point ⟨x, y⟩.
 * The search is started at the specified point i. If i is not specified, it defaults to zero.
 */
find(x: number, y: number, i?: number): number;
```
[DefinitelyTyped Definition](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/d3-delaunay/index.d.ts#L59-L63)

Furthermore, the actual `d3-delaunay` implementation is as follows: `if ((x = +x, x !== x) || (y = +y, y !== y)) return -1;` checks if `x` and `y` are numbers, and returns `-1` if they are not. However, since this Demo is adopting TypeScript, I believe there should be no issue as long as it adheres to TypeScript's type checking.
```js
find(x, y, i = 0) {
  if ((x = +x, x !== x) || (y = +y, y !== y)) return -1;
  const i0 = i;
  let c;
  while ((c = this._step(i, x, y)) >= 0 && c !== i && c !== i0) i = c;
  return c;
}
```
[d3-delaunay Implementation](https://github.com/d3/d3-delaunay/blob/main/src/delaunay.js#L131-L137)

Additionally, this `delaunay.find` is already used as follows in the demo of visx-delaunay-triangulation, and there isn’t a check for undefined there.
```tsx
const closest = delaunayDiagram.find(point.x - margin.left, point.y - margin.top);
setHoveredId(data[closest].id);
```
[visx-delaunay-triangulation Demo](https://github.com/airbnb/visx/blob/master/packages/visx-demo/src/sandboxes/visx-delaunay-triangulation/Example.tsx#L70-L71)